fluentui_others/src/main/java/com/microsoft/fluentui/vNext/widget/ButtonView.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+package com.microsoft.fluentui.vNext.widget
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatButton
+import com.microsoft.fluentui.R
+import com.microsoft.fluentui.generator.ButtonSize
+import com.microsoft.fluentui.generator.ButtonStyle
+import com.microsoft.fluentui.generator.resourceProxies.ButtonTokensSystem
+import com.microsoft.fluentui.util.FieldUpdateListener
+import com.microsoft.fluentui.vNext.persona.AvatarView
+
+/**
+ * [ButtonView] is a custom AppCompatButton that uses token system to set predefined button style
+ */
+open class ButtonView : AppCompatButton {
+
+    var button: Button
+        set(value) {
+            field = value
+            setValues()
+        }
+
+    private val tokenSystem: ButtonTokensSystem
+
+    @JvmOverloads
+    constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(context, attrs, defStyleAttr) {
```

Comment:
Please wrap the context with FluentUIContextThemeWrapper

Reply:
done